packages/enzyme-utilities/src/index.ts

```diff
@@ -49,6 +49,8 @@ export function trigger(wrapper: AnyWrapper, keypath: string, ...args: any[]) {
     }
   });
 
+  updateRoot(wrapper);
```

Comment:
baha, this makes a tonne of sense. Would you mind contributing it to https://github.com/Shopify/preact-testing as well? :P

Reply:
You already do it 🎉: https://github.com/Shopify/preact-testing/blob/f6025a84ba067b7e8def28383ef6d2739162d21f/src/root.tsx#L172